src/app/formFields/DynamicFormField.tsx

```diff
@@ -1,86 +1,57 @@
 import { FormField as FormFieldType } from '@bigcommerce/checkout-sdk';
 import { FieldProps } from 'formik';
-import React, { memo, useCallback, useMemo, FunctionComponent } from 'react';
+import React, { memo, useCallback, useMemo, FunctionComponent, ReactNode } from 'react';
 
 import { TranslatedString } from '../locale';
 import { FormField, Label } from '../ui/form';
 
-import { getFormFieldInputId, getFormFieldLegacyName } from './getFormFieldInputId';
 import CheckboxGroupFormField from './CheckboxGroupFormField';
 import DynamicFormFieldType from './DynamicFormFieldType';
 import DynamicInput from './DynamicInput';
 
-export interface AddressKeyMap<T = string> {
```

Comment:
I suggest that instead of having a top-level `formFields` module, maybe a better place to place the `DynamicFormField` component is `ui/form`. Otherwise, we'll end up with two quite similar modules as both contain generic form components. It can cause confusion in the future in terms of where to put things.